projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+  "block.minecraft.banner.rats.rat.orange": "橙色老鼠",
+  "block.minecraft.banner.rats.rat.pink": "粉红色老鼠",
+  "block.minecraft.banner.rats.rat.purple": "紫色老鼠",
+  "block.minecraft.banner.rats.rat.red": "红色老鼠",
+  "block.minecraft.banner.rats.rat.white": "白色老鼠",
+  "block.minecraft.banner.rats.rat.yellow": "黄色老鼠",
+  "block.minecraft.banner.rats.rat_and_crossbones.black": "黑色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.blue": "蓝色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.brown": "棕色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.cyan": "青色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.gray": "灰色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.green": "绿色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.light_blue": "淡蓝色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.light_gray": "淡灰色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.lime": "黄绿色老鼠和交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.magenta": "品红色老鼠和交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.orange": "橙色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.pink": "粉红色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.purple": "紫色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.red": "红色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.white": "白色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.yellow": "黄色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_sickle.black": "黑色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.blue": "蓝色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.brown": "棕色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.cyan": "青色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.gray": "灰色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.green": "绿色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.light_blue": "淡蓝色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.light_gray": "淡灰色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.lime": "黄绿色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.magenta": "品红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.orange": "橙色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.pink": "粉红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.purple": "紫色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.red": "红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.white": "白色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.yellow": "黄色老鼠和镰刀",
+  "block.rats.air_raid_siren": "防空警报",
+  "block.rats.auto_curdler": "凝乳台",
+  "block.rats.auto_curdler.desc0": "自动化奶酪生产",
+  "block.rats.auto_curdler.desc1": "§o“奶进，奶酪出”",
+  "block.rats.black_marbled_cheese": "黑色大理石纹奶酪",
+  "block.rats.block_of_blue_cheese": "蓝纹奶酪块",
+  "block.rats.block_of_cheese": "奶酪块",
+  "block.rats.block_of_nether_cheese": "熔岩奶酪块",
+  "block.rats.brain_block": "脑方块",
+  "block.rats.cauldron_blue_cheese": "蓝纹奶酪炼药锅",
```

```suggestion
  "block.rats.cauldron_blue_cheese": "装有蓝纹奶酪的炼药锅",
```
原版句式。下同，不一一指出
[Minecraft Wiki:译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96)